### PR TITLE
Fix the approval steps ordering

### PR DESCRIPTION
**Pull request by rememberlenny**
# What

The Approval steps were loading out of order.

Fixing this.

screen shot 2016-04-27 at 11 49 55 am
screen shot 2016-04-27 at 11 49 44 am

### CODE REVIEW

File: app/views/proposals/details/_approvals.html.haml [hunk 1]
```diff
@@ -7,7 +7,7 @@
             Approvals
         .content-content.column
           - index = 0
-          - proposal.steps.each do |step|
+          - proposal.steps.order('position ASC').each do |step|
```
Comment (pkarman) [inline app/views/proposals/details/_approvals.html.haml]: I believe this is already available as

```
proposal.individual_steps
```

which should be in position order.

File: app/views/proposals/details/_summary.html.haml [hunk 2]
```diff
@@ -4,10 +4,10 @@
       Request Status 
       %span.status-value
         = proposal.capitalized_detailed_status
-    - if proposal.status == "completed" && proposal.steps.last.completed_at
+    - if proposal.status == "completed"
       %span.completed-time
         on 
-        = proposal.steps.last.completed_at.strftime("%b %d, %Y at %l:%M%P")
+        = proposal.steps.order('position ASC').last.completed_at.strftime("%b %d, %Y at %l:%M%P")
```
Comment (pkarman) [inline app/views/proposals/details/_summary.html.haml]: try `proposal.individual_steps` to avoid needing the explicit sort.

**Comment by rememberlenny** [comment]
@pkarman Thanks for that. I thought that should have existed.

**Comment by pkarman** [comment]
code looks fine -- I'm happy to merge once circleci is 💚

**Comment by rememberlenny** [comment]
@pkarman Clear!

**Pull request merged**